src/main.cpp

```diff
@@ -429,8 +431,9 @@ void setup() {
     //  Build shader;
     //
     if (iFrag != -1) {
+        fragPath = &files[iFrag].path;
```

Comment:
I have some project that use the `#include "something.glsl"` recursively. That is shaders than call `snip1.glsl` that call `snip1_1.glsl` etc... have you test it in those cases? I'm afraid of braking something here